utils/R/convert.input.R

```diff
@@ -9,7 +9,7 @@
 convert.input <- function(input.id, outfolder, formatname, mimetype, site.id, start_date, 
                           end_date, pkg, fcn, con = con, host, browndog, write = TRUE, 
                           format.vars, overwrite = FALSE, exact.dates = FALSE, 
-                          allow.conflicting.dates = TRUE, insert.new.file = FALSE,...) {
+                          allow.conflicting.dates = TRUE, insert.new.file = FALSE, met = NULL,...) {
```

Comment:
I've got to ask about the choice of name here -- it appears that this arg is only used in a call to db.input.check where it's passed as ```pattern = met```. So why not call this arg ```pattern``` as well? If I use convert.input to process initial condition data, it's pretty unintuitive to have an argument called 'met', especially when the argument itself doesn't actually have anything to do with met, but is just the 'text to search for in the file name' when looking for existing files.